various small fixes (#264)
python-trio/flake8-async: docs/rules.rst

_`ASYNC113` : start-soon-in-aenter
    Using :meth:`~trio.Nursery.start_soon`/:meth:`~anyio.abc.TaskGroup.start_soon` in ``__aenter__`` doesn't wait for the task to begin.

>     This will only warn about built-in functions and those listed in _`ASYNC114`.

Reviewer: ```suggestion
    This will only warn about functions listed in _`ASYNC114` or known from Trio or AnyIO.
```
